ompi/dpm/dpm.c

```diff
@@ -1126,7 +1127,7 @@ int ompi_dpm_dyn_finalize(void)
     ompi_communicator_t *comm=NULL;
 
     if (1 <ompi_comm_num_dyncomm) {
-        objs = (ompi_dpm_disconnect_obj**)malloc(ompi_comm_num_dyncomm*
```

Comment:
Not clear what is the purpose of the +1.

Reply:
I agree, removed it. I think he was trying to make it consistent with line 1144, but I think that was also incorrect..My understanding is if j != ompi_comm_num_dyncomm it should be an error, so I have no idea where that +1 is coming from. I assume it's a miscount/overlook.